src/engine/moods.ts

```diff
@@ -71,11 +71,19 @@ function getRelevantEffects(): { [modifier: string]: Effect[] } {
   if (
     have($item`Clan VIP Lounge key`) &&
     !underStandard() &&
-    (!get("_olympicSwimmingPool") || have($effect`Silent Running`)) &&
+    (get("_photoBoothEffects",0) < 3 || have($effect`Wild and Westy!`)) &&
     ((have($item`ninja rope`) && have($item`ninja carabiner`) && have($item`ninja crampons`)) ||
       step("questL08Trapper") >= 3)
   )
-    result["-combat"].push($effect`Silent Running`);
+    result["-combat"].push($effect`Wild and Westy!`);
+
+    if (
+      have($item`Clan VIP Lounge key`) &&
+      !underStandard() &&
+      (!get("_olympicSwimmingPool") || have($effect`Silent Running`)) &&
+      ((have($item`ninja rope`) && have($item`ninja carabiner`) && have($item`ninja crampons`)) ||
+        step("questL08Trapper") >= 3)
+    )
```

Comment:
You accidentally deleted the `result["-combat"].push($effect`Silent Running`);` line.